### PR TITLE
fix bug 'file:'; do not transfer file for condor at tamsa

### PR DESCRIPTION
**Pull request by sansan9401**
* Do not transfer files for condor job at tamsa
  * this is mush faster
  * chmod 755
* Fix bug: remove some duplication of "file:"

### CODE REVIEW

**Comment by sihyunjeon** [comment]
You specified config.JobType.numCores=8 in the crab configuration file but process.options.numberOfThreads=cms.untracked.uint32(4). Please make sure the two parameters are consistent and have the same value (or they are both missing) in the crab configuration file and in the CMSSW PSet (TESTCRAB.py)

Also found one more bug for CRABJOB mode.
Will check whether Hyonsan's fix works in tamsa, cms-snu, knu, kisti and then merge.

File: packages/FullSimulation/MiniAOD/make_MiniAOD.py [hunk 1]
```diff
@@ -164,20 +164,24 @@ def exit_runhosterr(runmode,hostname):
...
     submitshellfile.write("cd "+cwd+"/"+runmode+"/"+sample_name+"/run"+str(ijob)+"\n")
     if (runmode == "MULTICORE"):
-      submitshellfile.write("chmod 755 "+cwd+"/"+runmode+"/"+sample_name+"/run"+str(ijob)+"/run"+str(ijob)+".sh\n")
       submitshellfile.write("nohup "+cwd+"/"+runmode+"/"+sample_name+"/run"+str(ijob)+"/run"+str(ijob)+".sh &\n")
     elif (runmode == "CLUSTER"):
       os.system("cp skeleton/condor.jds "+runmode+"/"+sample_name+"/run"+str(ijob))
       os.system("sed -i 's|###CONDORRUN|executable = run"+str(ijob)+".sh|g' "+runmode+"/"+sample_name+"/run"+str(ijob)+"/condor.jds")
       os.system("sed -i 's|###JOBBATCHNAME|+JobBatchName=\""+sample_name+"\"|g' "+runmode+"/"+sample_name+"/run"+str(ijob)+"/condor.jds")
       submitshellfile.write("condor_submit condor.jds\n")
+      if os.getenv("HOSTNAME") in ["tamsa1","tamsa2"]:
```
Comment (sihyunjeon) [inline packages/FullSimulation/MiniAOD/make_MiniAOD.py]: Can you change these lines to "if (use_SNU == True):" instead?
CLUSTER & SNU = tamsa server

Comment (sansan9401) [reply packages/FullSimulation/MiniAOD/make_MiniAOD.py]: done

**Comment by sihyunjeon** [comment]
Worked in tamsa, cms-snu, and knu

**Pull request merged**